npps4/webui/endpoints/list_users.py

```diff
@@ -0,0 +1,18 @@
+import fastapi
+import sqlalchemy
+
+from ... import idol
+from ...db import main
+from .. import template
+from ...app import webui
```

Comment:
I forgot to mention that the alphabetical order sorting for `from <a> import <b>` notation, the first order should be `<a>` then `<b>`. It's not your fault, but I'll update the CONTRIBUTING.md.